lib/manager/helmv3/artifacts.ts

```diff
@@ -21,6 +22,15 @@ async function helmUpdate(manifestPath: string): Promise<void> {
       HELM_EXPERIMENTAL_OCI: '1',
     },
   };
+  const cmd = [
+    `helm dependency update ${quote(getSubDirectory(manifestPath))}`,
+  ];
+
+  if (aliases) {
+    Object.entries(aliases).forEach(([alias, url]) =>
+      cmd.push(`helm repo add ${quote(alias)} ${quote(url)}`)
```

Comment:
Don't you need these added before the `helm dependency update` command is run? Your snapshot now is `helm dependency update '' && helm repo add stable the_stable_url && helm repo add repo1 the_repo1_url`

Reply:
you're right... it's almost the purpose of this PR ...